pilot/pkg/serviceregistry/kube/controller/namespacecontroller_test.go

```diff
@@ -32,33 +35,41 @@ import (
 
 func TestNamespaceController(t *testing.T) {
 	client := kube.NewFakeClient()
+	lister := createFakeLister(client)
```

Comment:
should use `client.Informer()...`. There is already a `SharedInformerFactory` in the client, but we are creating a second one

Reply:
thanks pointing out, fixed

Reply:
It seems to still be doing the same?

Reply:
could you check again for latest code? and need your help to add label release-notes-none

Reply:
it looks like it changed a bit but not fully as expected. Don't call NewSharedInformerFactory. The kube client already has one created we should use. Just call kube.Informers() (or similar, I forget the exact name) and you should be good to go.

Reply:
@howardjohn not sure if you are seeing the latest code, but if you look at the latest push, NewSharedInformerFactory is not called in namespacecontroller_test.go, the ut is fully leveraging fake client.
the function is needed in security/pkg/k8s/configutil_test.go, since there is a createConfigMapDisabledClient being created in the ut code.

Reply:
I see, I think I was looking at the configmap util one. Thanks!

Reply:
@howardjohn all uts passed